tools/docker-compose/supervisor.conf

```diff
@@ -95,6 +95,16 @@ redirect_stderr=true
 stdout_events_enabled = true
 stderr_events_enabled = true
 
+[program:minikube-credentials]
+command = make minikube-credentials
```

Comment:
This feels very wrong 😄 Let's chat more tomorrow about what you're trying to accomplish here.

Reply:
Yes, it does :) I actually took a different approach which I believe will simplify the process and work better. I'll update the PR shortly